### PR TITLE
[GUI] [Trivial] Changes primarily to written content style. Removal/updates of Zerocoin privacy mentions.

### PR DESCRIPTION
**Pull request by StrontiumZ38**
These changes are primarily aimed at unifying the written content style within the wallet, which has been very inconsistent. There are also some changes to written content regarding Zerocoin and privacy to better reflect the state the protocol is now in within the Veil Core wallet Lastly, there is a comment on an anomaly on a string in the automint settings. This can largely be disregarded as it's working anyway, but perhaps unintentional.

These written style changes are not comprehensive, and will need to be revised along with instructions/explanations as RingCT becomes the new standalone protocol in the future.

### CODE REVIEW

File: src/qt/veil/forms/settingsfaq05.ui [hunk 1]
```diff
@@ -38,10 +38,10 @@ color:#707070;</string>
 To spend Basecoin veil, follow these instructions:
 
     1. Navigate to the Send tab.
-    2. Click the Coin Control button.
+    2. Click the Coin control button.
     3. Select all Basecoin UTXO inputs and send a transaction.
 
-All Basecoin transactions are anonymized with RingCT. Although RingCT transactions have an anonymity set size of 11 versus Zerocoin’s thousands, it prevents Basecoin from compromising the privacy of Zerocoin transactions.”					</string>
+Alternatively, the console command 'semdtoaddress' will allow for the spending of Basecoin veil to a specified address.”					</string>
```
Comment (JSKitty) [inline src/qt/veil/forms/settingsfaq05.ui]: Typo: 'semdtoaddress'

File: src/qt/veil/forms/settingsmintingauto.ui [hunk 2]
```diff
@@ -218,7 +218,7 @@ font-size:16px;</string>
 font-size:16px;</string>
         </property>
         <property name="text">
-         <string>100000</string>
+         <string>100000</string> // What is this about?
```
Comment (JSKitty) [inline src/qt/veil/forms/settingsmintingauto.ui]: Is this comment syntax correct...? Reference: [The .ui File Format](https://doc.qt.io/archives/3.3/designer-manual-16.html#1-4)

Comment (StrontiumZ38) [reply src/qt/veil/forms/settingsmintingauto.ui]: Can't comment off line that way? Removed.

File: src/qt/veil/forms/settingsfaq05.ui (hunk 1 above)
Comment (StrontiumZ38) [reply src/qt/veil/forms/settingsfaq05.ui]: Nice catch, thanks!

**Comment by StrontiumZ38** [comment]
@codeofalltrades I believe squashing needs Gitian software? I've just been working on GitHub as I couldn't get Gitian working for me last time I tried.

**Comment by crypt0-fusi0n** [comment]
Hmmm, this one looks like a partial pass at the moment...

Some of the changes are reflected in the UI, while others are not. This includes changes made to the same file, so it's possible that the qt wallet isn't reading resource strings from the same file?

**Comment by crypt0-fusi0n** [comment]
image

Here's an example... "**Unlock Wallet for Staking**" appears in multiple src files...

image

**veil_en.ts**, **settingswidget.ui**, **unlockpassworddialog.ui** all have "Unlock wallet for staking" but it's ultimately overwritten by **settingswidget.cpp** and **unlockpassworddialog.cpp**

**Comment by crypt0-fusi0n** [comment]
Another example...

image

res strings

**Comment by codeofalltrades** [comment]
I created a new issue with remaining bugs. Considering this passed.

**Pull request merged**